core/src/main/scala/org/http4s/multipart/MultipartParser.scala

```diff
@@ -274,7 +266,7 @@ object MultipartParser {
       c: Chunk[Byte]
   ): (Int, Stream[F, Byte], Stream[F, Byte]) = {
     val ixx = i - currState
-    if (middleChunked || state == 0) {
```

Comment:
state == 0 condition was removed because it is not longer necessary: this case will be picked up as a `middleChunked` case, now always.